sdcm/cluster.py

```diff
@@ -1660,6 +1660,13 @@ def stop_nemesis(self, timeout=10):
         self.nemesis_threads = []
         self.log.debug('Stop nemesis end')
 
+    def _experimental(self):
+        for cmd in ('stress_cmd', 'stress_cmd_1'):
+            stress_cmd = self.params.get(cmd)
+            if stress_cmd and 'counter_' in stress_cmd:
+                return True
+        return False
+
 
```

Comment:
changed, we just need to add the following line to tests yaml:
experimental: 'true'